.github/workflows/virtualbox_image.yml

```diff
@@ -0,0 +1,110 @@
+name: Create VirtualBox disk image from Dockerfile
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  release:
+    types:
+      - published
+  push:
+    paths:
+      - 'docker/*'
+      - '.github/workflows/virtualbox_image.yml'
+
+env:
+  VM_PASSWORD: temppwd
+  DOCKER_IMAGE_BASENAME: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3
+  DOCKER_IMAGE_TAG: :refs_heads_${{ github.ref_name }}-bk0
+
+jobs:
+  pyne_image_build:
+    runs-on: ubuntu-latest
+    outputs:
+      image_tag: ${{ steps.output_tag.outputs.image_tag }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build PyNE docker image
+        uses: firehed/multistage-docker-build-action@v1
+        id: build_pyne
+        with:
+          repository: ${{ env.DOCKER_IMAGE_BASENAME }}
+          stages: base_python, moab, dagmc, openmc
+          server-stage: pyne
+          quiet: false
+          tag-latest-on-default: false
+          dockerfile: docker/ubuntu_22.04-dev.dockerfile
+          build-args: build_hdf5=hdf5-1_12_0, build_pyne=YES
```

Comment:
The `repository` doesn't match for this hdf5 `build-arg`

We should probably use the one without `hdf5-1_12_0` since it doesn't build openmc in the docker publish workflow.